Project SF/Views/Tabs/Settings/ProfileSettingsView.swift

```diff
@@ -1,43 +0,0 @@
-//
-//  ProfileSettingsView.swift
-//  Project SF
-//
-//  Created by Roman Esin on 11.07.2020.
-//
-
-import SwiftUI
-
-struct ProfileSettingsView: View {
-
-    @State var username = ""
-
-    @State var isShowingAlert = false
-
-    var body: some View {
-        VStack {
-            Form {
-                TextField("Enter your name", text: $username) { (didChange) in
-                    print(didChange)
-                } onCommit: {
-                    guard !username.isEmpty else {
-                        isShowingAlert = true
-                        return
-                    }
-                }
-            }
-        }
-        .alert(isPresented: $isShowingAlert) {
-            Alert(title: Text("Please, fill in your name"), message: nil, dismissButton: .default(Text("Ok")))
-        }
-    }
-
-    init() {
-        username = UserDefaults.standard.string(forKey: "username") ?? ""
```

Comment:
Use the new @AppStorage instead of init with UserDefaults.

Reply:
Ok will fix that in a sec